crates/core/src/mock/fixtures/requests/kakarot_getCode.json

```diff
@@ -0,0 +1,13 @@
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "starknet_call",
+  "params": [
+    {
+      "contract_address": "0xd90fd6aa27edd344c5cbe1fe999611416b268658e866a54265aaf50d9cf28d",
```

Comment:
I don't understand what is this request, why it's named `kakarot_getCode` while there is no calldata?

Reply:
i think it is the evm_contract.bytecode() to fetch the bytecode of an evm contract on starknet, deployed thru kakarot

Reply:
indeed the selector `0x2f22d9e1ae4a391b4a190b8225f2f6f772a083382b7ded3e8d85743a8fcfdcd` is the selector for bytecode